sdk/src/claim.rs

```diff
@@ -1827,13 +1828,15 @@ impl Claim {
     /// Verify claim signature, assertion store and asset hashes
     /// claim - claim to be verified
     /// asset_bytes - reference to bytes of the asset
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Clippy is not wrong here. Let's explore (again in a future PR) whether we can reduce # of arguments.